packages/web-components/src/anchor/anchor.stories.ts

```diff
@@ -0,0 +1,213 @@
+import { html } from '@microsoft/fast-element';
+import type { Args, Meta } from '@storybook/html';
+import { renderComponent } from '../helpers.stories.js';
+import type { Anchor as FluentAnchor } from './anchor.js';
+import { AnchorAppearance, AnchorShape, AnchorSize } from './anchor.options.js';
+import './define.js';
+
+type AnchorStoryArgs = Args & FluentAnchor;
+type AnchorStoryMeta = Meta<AnchorStoryArgs>;
+
+const storyTemplate = html<AnchorStoryArgs>`
+  <fluent-anchor
+    href="${x => x.href}"
+    appearance="${x => x.appearance}"
+    shape="${x => x.shape}"
+    size="${x => x.size}"
+    ?disabled="${x => x.disabled}"
+    ?disabled-focusable="${x => x.disabledFocusable}"
+    ?icon-only="${x => x.iconOnly}"
+    ?icon="${x => x.icon}"
```

Comment:
After the latest changes in `fluent-button`, there is no `icon` attribute, right? Remove it everywhere.